app/views/course/discussion/posts/_form.html.slim

```diff
@@ -0,0 +1,5 @@
+div.post-form style='display: none'
```

Comment:
Do you need to add a comment here for `display: none`, since the intention is to disable this if the browser doesn't support javascript?